MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -181,6 +184,23 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         let customPermission = self.miniAppKeyStore.getCustomPermissions(forMiniApp: self.miniAppId).filter { $0.permissionName == customPermissionType }
         return customPermission[0].isPermissionGranted.boolValue
     }
+
+    func setScreenOrientation(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.action else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+            return
+        }
+        if !requestParamValue.isEmpty {
+            guard let info = MiniAppInterfaceOrientation(rawValue: requestParamValue) else {
+                self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+                return
+            }
+            delegate?.didOrientationChanged(orientation: info.orientation)
+            UIViewController.attemptRotationToDeviceOrientation()
```

Comment:
Sorry, I missed something here. We need to send a `success` callback to the mini app. The Android SDK sends success after then orientation was successfully set.

It looks like there isn't a way to detect if `attemptRotationToDeviceOrientation` was successful? So maybe it's okay just to send success immediately after calling `attemptRotationToDeviceOrientation`?

Reply:
Oh, sorry. I thought Android isn't sending any success response, I looked at few android comments. No probs, I will make changes in another PR 👍